source/test/TestApp/TestApp.cpp

```diff
@@ -104,6 +107,12 @@ namespace PlayFabUnit
         testRunner.Add(pfMultiUserInstanceTest);
 #endif
```

Comment:
This is the end of the #if !defined(DISABLE_PLAYFABCLIENT_API) block.
(Maybe we should comment-label these endif's for clarity)